<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EditMeasuredDataDialog</class>
 <widget class="QDialog" name="EditMeasuredDataDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>300</width>
    <height>400</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>300</width>
    <height>400</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>300</width>
    <height>400</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Edit Measured Data</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <property name="sizeGripEnabled">
   <bool>false</bool>
  </property>
  <widget class="QLabel" name="label_10">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>460</y>
     <width>55</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>57</x>
     <y>304</y>
     <width>161</width>
     <height>31</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBoxShowLegend">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>30</y>
     <width>111</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Show Legend </string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBoxShowFullPath">
   <property name="geometry">
    <rect>
     <x>39</x>
     <y>64</y>
     <width>220</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Show full paths in legend</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>21</x>
     <y>139</y>
     <width>241</width>
     <height>146</height>
    </rect>
   </property>
   <property name="title">
    <string>Modify</string>
   </property>
   <widget class="QComboBox" name="comboBoxDataSets">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>40</y>
      <width>85</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>40</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Data Set</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButtonSetColor">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>70</y>
      <width>85</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Set  Co&amp;lor</string>
    </property>
    <attribute name="buttonGroup">
     <string notr="true">buttonGroup</string>
    </attribute>
   </widget>
   <widget class="QRadioButton" name="radioButtonRemoveSet">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>99</y>
      <width>76</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Remove</string>
    </property>
    <attribute name="buttonGroup">
     <string notr="true">buttonGroup</string>
    </attribute>
   </widget>
   <widget class="QLabel" name="labelColor">
    <property name="geometry">
     <rect>
      <x>154</x>
      <y>74</y>
      <width>50</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
  </widget>
  <widget class="QCheckBox" name="checkBoxDataPoints">
   <property name="geometry">
    <rect>
     <x>38</x>
     <y>103</y>
     <width>166</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Show data points only</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>EditMeasuredDataDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>190</x>
     <y>585</y>
    </hint>
    <hint type="destinationlabel">
     <x>203</x>
     <y>308</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>EditMeasuredDataDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>190</x>
     <y>585</y>
    </hint>
    <hint type="destinationlabel">
     <x>203</x>
     <y>308</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>getFileName()</slot>
 </slots>
 <buttongroups>
  <buttongroup name="buttonGroup"/>
 </buttongroups>
</ui>
